pyartifactory/models/build.py

```diff
@@ -0,0 +1,252 @@
+"""
+Definition of all build models.
+"""
+from __future__ import annotations
+
+from typing import Dict, List, Optional
+
+from pydantic import BaseModel, Field
+
+
+class BuildProperties(BaseModel):
+    started: Optional[str] = None
+    diff: Optional[str] = None
+    project: Optional[str] = None
+
+    # Method to convert model to query string
+    def to_query_string(self) -> str:
+        # Create a list of key=value pairs for all non-None fields
+        properties = [f"{key}={value}" for key, value in self.model_dump(exclude_none=True).items()]
+
+        return "?" + "&".join(properties) if properties else ""
+
+
+class SimpleBuild(BaseModel):
+    """Models an artifactory single build."""
+
+    uri: str
+    lastStarted: str
+
+
+class BuildListResponse(BaseModel):
+    """Models all artifactory builds."""
+
+    uri: str = ""
+    builds: Optional[List[SimpleBuild]] = None
+
+
+class Run(BaseModel):
+    """Models an artifactory single build run."""
+
+    uri: str
+    buildsNumbers: str
```

Comment:
From what I see in the [Artifactory API](https://jfrog.com/help/r/jfrog-rest-apis/build-runs), this params should be `started` instead